webapp/src/components/disease/index.js

```diff
@@ -1,17 +1,19 @@
+/* eslint-disable react/no-multi-comp */
+
 import React, { Component } from 'react';
 import FlybaseDataGrid from 'react-flybase-datagrid';
 
-function getHeaders(){
+const doLink = (name, row) => (
```

Comment:
Just a thought. This might be a good one to write as a functional Reaction component, for the benefit of props validation. A bit on functional (stateless) component https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components